js/core/activity-feedback-data.js

```diff
@@ -97,8 +99,9 @@ export function calculateStudentScores(state, questions) {
 
 export function getComputedMaxScore(questions) {
   return questions
+    .filter(question => question.get('scoreEnabled'))
     .map(question  => question.get('maxScore') || 0)
-    .reduce( (total, score) => total + score)
+    .reduce( (total, score) => total + score) || 0
```

Comment:
Just wondering which case is this `|| 0` handling? If scores are all 0, it should return 0 anyway. If we call `reduce` on an empty array (e.g. due to filtering above), `reduce` will throw an error if we skip its second argument (initial value). If that was your intention, we'd rather need to use `.reduce( (total, score) => total + score), 0 )`. It handles empty array case.

Reply:
Thanks for this. I will make this change.